README.md

```diff
@@ -110,6 +108,10 @@ Build and push the HCO's registry image.
 export IMAGE_REGISTRY=<image_registry>
 export REGISTRY_NAMESPACE=<container_org>
 export CONTAINER_TAG=example
+export HCO_OPERATOR_IMAGE=$IMAGE_REGISTRY/$REGISTRY_NAMESPACE/hyperconverged-cluster-operator:$CONTAINER_TAG
+
+# Image to be used in CSV manifests
+HCO_OPERATOR_IMAGE=$HCO_OPERATOR_IMAGE CSV_VERSION=$CSV_VERSION make build-manifests
```

Comment:
I don't think that `make bundleRegistry` on the next line is still accurate.
Now we have to build the bundle image and then the index image and use the index image for the catalogSource or use `operator-sdk run bundle <bundle-image>`

Reply:
> Now we have to build the bundle image and then the index image and use the index image for the catalogSource

Do we have any make target for it? Usually projects initialized with `operator-sdk init` have the targets `bundle` and `bundle-build` that build the index image. It's happening in `hack/build-index-image.sh`, I think:
```shell
$ podman build -t "${BUNDLE_IMAGE_NAME}" -f bundle.Dockerfile --build-arg "VERSION=${CURRENT_VERSION}" .
```
But I don't see it being run through any make target. Should it have a make target or do you suggest I put this podman command here and elaborate?

Reply:
Yes, we use `hack/build-index-image.sh` but we lack a `make target` calling it.
We need to decide if we want to suggest this path or simply suggest to use `operator-sdk run bundle <bundle-image>` to keep it simpler.
I'm for the second one.

Reply:
> I'm for the second one.

Me too. But don't we need to generate the bundle first before being able to run it? So something like: `operator-sdk generate bundle` and then `operator-sdk run bundle <bundle-image>`?

Reply:
we have our dockerfile as `bundle.Dockerfile`, is `operator-sdk generate bundle` correctly picking it up?
if not we should simply recommend to build it first, and then execute with `operator-sdk run bundle <bundle-image>`

Reply:
OK, so I ran the following commands, but I'm getting error at the last step:
```sh
$ operator-sdk generate bundle --input-dir deploy/index-image/ --output-dir /tmp/bundle
$ operator-sdk bundle validate /tmp/bundle
$ docker build -f deploy/index-image/bundle.Dockerfile -t $IMAGE_REGISTRY/$REGISTRY_NAMESPACE/hco-index:$CONTAINER_TAG deploy/index-image
$ docker push $IMAGE_REGISTRY/$REGISTRY_NAMESPACE/hco-index:$CONTAINER_TAG deploy/index-image
$ operator-sdk bundle validate $IMAGE_REGISTRY/$REGISTRY_NAMESPACE/hco-index:$CONTAINER_TAG

# error
$ operator-sdk run bundle $IMAGE_REGISTRY/$REGISTRY_NAMESPACE/hco-index:$CONTAINER_TAG
INFO[0011] Creating a File-Based Catalog of the bundle "quay.io/dharmit_rh/hco-index:latest"
INFO[0013] Generated a valid File-Based Catalog
FATA[0013] Failed to run bundle: create catalog: error creating registry pod: error building registry pod definition: configMap error: error updating ConfigMap: error creating ConfigMap: ConfigMap "community-kubevirt-hyperconverged-catalog-configmap-partition-2" is invalid: []: Too long: must have at most 1048576 bytes
```

Have you or @nunnatsa tried this earlier? Do we know if it worked? I'm not 100% sure, but I vaguely remember some discussion around the long name issue.